app/models/podcast_import.rb

```diff
@@ -126,12 +128,12 @@ def create_podcast
     podcast_attributes[:explicit] = feed.itunes_explicit
     podcast_attributes[:new_feed_url] = feed.itunes_new_feed_url
     podcast_attributes[:path] ||= feed.feedburner_name
+    podcast_attributes[:feedburner_url] ||= feed.feedburner_url
+    podcast_attributes[:url] ||= feed.feedburner_url
 
-    podcast_attributes[:author] = person(feed.itunes_author)
+    podcast_attributes[:author] = feed.author ? person(feed.author) : person(feed.itunes_author)
```

Comment:
I know we talked about this, and I probably said this wrong, but there isn't a `feed.author`